pcap/pcap.h

```diff
@@ -37,6 +37,10 @@
 
 #include <pcap/export-defs.h>
 
+#if defined(WIN32) && _MSC_VER < 1900
```

Comment:
Hi @guyharris ,

In https://github.com/the-tcpdump-group/libpcap/blob/master/pcap-int.h#L412-L422,

``pcap_snprintf`` is also defined as ``snprintf`` if ``HAVE_SNPRINTF`` is defined (which is actually defined in the projecr file). And my VS 2013 doesn't define ``snprintf``, so we still need to face this issue. How do you suggest to solve it then?